content/docs/guides/file-sd.md

```diff
@@ -0,0 +1,117 @@
+
+```bash
+curl http://localhost:9100/metrics
+```
+
+The metrics output should look something like this:
+
+```
+# HELP go_gc_duration_seconds A summary of the GC invocation durations.
+# TYPE go_gc_duration_seconds summary
+go_gc_duration_seconds{quantile="0"} 0
+go_gc_duration_seconds{quantile="0.25"} 0
+go_gc_duration_seconds{quantile="0.5"} 0
+...
+```
+
+## Installing, configuring, and running Prometheus
+
+Like the Node Exporter, Prometheus is a single static binary that you can install via tarball. [Download the latest release](/download#prometheus) for your platform and untar it:
+
+```bash
+wget https://github.com/prometheus/prometheus/releases/download/v*/prometheus-*.*-amd64.tar.gz
+tar xvf prometheus-*.*-amd64.tar.gz
+cd prometheus-*.*
+```
+
+The untarred folder contains a `prometheus.yml` configuration file. Replace the current contents of that file with this:
+
+```yaml
+scrape_configs:
+- job_name: 'node'
+  file_sd_configs:
+  - files:
+    - 'targets.json'
+```
+
+This configuration specifies that there is a job called `node` (for the Node Exporter) that retrieves host and port information for Node Exporter instances from a `targets.json` file.
+
+Now create that `targets.json` file and add this content to it:
+
+```json
+[
+  {
+    "labels": {
+      "job": "node"
+    },
+    "targets": [
+      "localhost:9100"
+    ]
+  }
+]
+```
+
+This configuration specifies that there is a `node` job with one target: `localhost:9100`.
+
+Now you can start up Prometheus:
+
+```bash
+./prometheus
+```
+
+If Prometheus has started up successfully, you should see a line like this in the logs:
+
+```
+level=info ts=2018-08-13T20:39:24.905651509Z caller=main.go:500 msg="Server is ready to receive web requests."
+```
+
+## Exploring the discovered services' metrics
+
+With Prometheus up and running, you can explore metrics exposed by the `node` service using the Prometheus [expression browser](/docs/visualization/browser). If you explore the [`up{job="node"}`](http://localhost:9090/graph?g0.range_input=1h&g0.expr=up%7Bjob%3D%22node%22%7D&g0.tab=1) metric, for example, you can see that the Node Exporter is being appropriately discovered.
+
+## Changing the targets list dynamically
+
+When using Prometheus' file-based service discovery mechanism, the Prometheus instance will listen for changes to the file and automatically update the scrape target list, without requiring an instance restart. To demonstrate this, start up a second Node Exporter instance on port 9200. First navigate to the folder containing the Node Exporter binary and run this command in a new terminal window:
+
+```bash
+./node_exporter --web.listen-address=":9200"
+```
+
+Now modify the config in `targets.json` by adding the new Node Exporter instance to the `targets` list:
+
+```json
+[
+  {
+    "targets": [
+      "localhost:9100",
+      "localhost:9200"
```

Comment:
I'd suggest adding this as a second dict, rather than in here. It'll make it more obvious that you can have different labels.

I'd also mention that JSON and commas can be finicky, and that you should probably use a tool/library to generate your json.